.github/actions/Dockerfile

```diff
@@ -0,0 +1,63 @@
+
+FROM mcr.microsoft.com/powershell:latest
```

Comment:
Evaluate the ADO pipeline approach for running the arm-ttk in an agent and compare the performance metrics running in docker and running in ADO agent

Reply:
I did some research on how we can use the ADO pipeline but didn't got any working way. In our case we had to do custome validations and identify files and then run the arm-ttk.